test/e2e/specs/editor/various/copy-cut-paste.spec.js

```diff
@@ -582,7 +582,7 @@ test.describe( 'Copy/cut/paste', () => {
 		} );
 
 		await pageUtils.pressKeys( 'primaryAlt+v' );
-		expect( await editor.getBlocks() ).toMatchObject( [
+		await expect.poll( editor.getBlocks ).toMatchObject( [
```

Comment:
The `expect.poll` should be the preferred method for similar assertions. Not sure if the whole overhaul is worth the effort, so let's fix this case by case.